### DISCUSSION TITLE
FastLoginBungee

### DISCUSSION THREAD

**Discussion by LisooYT** [games647/FastLogin#1147] [Q&A]
I have a quick question when i use AuthMeBungee + FastLoginBungee then i added also FastLogin and Authme to my lobby server okay it works i setupped everythnig but i cant send any messages nor use commands (console commands work) when i join only authme login works after logging i cant chat nor use commands, im 100% sure its not lag

- I do not use any database do i need one?
if i need one is there a way to like make this without bungee options so like it will act as normal server?

**Comment by TuxCoding** [comment]
>I do not use any database do i need one?

Yes, you need to have MySQL or MariaDB instance. 

>but i cant send any messages nor use commands (console commands work) 

Please check the `proxy.log` for any errors.
